src/devtools/client/debugger/src/components/App.js

```diff
@@ -64,30 +63,36 @@ class Debugger extends Component {
   };
 
   getChildContext = () => {
-    return { shortcuts, l10n: L10N };
+    return { shortcuts: this.shortcuts, l10n: L10N };
   };
 
+  constructor(props) {
+    super(props);
+    this.shortcuts = new KeyShortcuts({ window, target: props.wrapper });
+  }
+
   componentDidMount() {
-    shortcuts.on("CmdOrCtrl+Shift+P", this.toggleSourceQuickOpenModal);
-    shortcuts.on("CmdOrCtrl+Shift+O", this.toggleFunctionQuickOpenModal);
-    shortcuts.on("CmdOrCtrl+P", this.toggleSourceQuickOpenModal);
```

Comment:
It would be nice if `cmd+p`, `cmd+o`, `cmd+shift+p` and `cmd+shift+o` remained global. perhaps others as well